input/fsh/who-cr-CompositionCaseReportBasic.fsh

```diff
@@ -0,0 +1,261 @@
+* category = $LNC#55751-2
+* encounter 1..1
+* identifier 1..1
+
+* section ^slicing.discriminator.type = #value
+* section ^slicing.discriminator.path = "code.coding.code"
+* section ^slicing.rules = #openAtEnd
+* section ^slicing.ordered = true
+* section ^slicing.description = "Slice of composition.section based on code"
+
+* section contains
+    patientInformation 1..1 and
+    clinicalStatus 1..1 and
+    exposureRisk 1..1
+
+* section[patientInformation].title = "patient information"
+* section[patientInformation].code = http://test.org/sectionCode#patientinformation
+* section[patientInformation].entry only Reference(WhoCrObservationAge or WhoCrObservationBirthSex)
+
+* section[clinicalStatus].title = "clinical status"
+* section[clinicalStatus].code = http://test.org/sectionCode#clinicalStatus
+* section[clinicalStatus].entry only Reference(WhoCrObservationLabTestDate or WhoCrObservationSymptoms or WhoCrObservationOnsetDate or WhoCrObservationComorbidities)
+
+* section[exposureRisk].title = "exposure risk"
+* section[exposureRisk].code = http://test.org/sectionCode#exposureRisk
+* section[exposureRisk].entry only Reference(WhoCrObservationHasTravelled)
+
+Profile: WhoCrEncounter
+Parent: Encounter
+Id: who-cr-encounter
+Title: "WHO CR Encounter"
+Description: "WHO Encounter for a case report"
+* location 1..1
+* reasonCode 1..*
+* reasonCode from WhoCrValueSetReasonForCovid19Testing
+
+Profile: WhoCrLocation
+Parent: Location
+Id: who-cr-location
+Title: "WHO CR Location"
+Description: "WHO Location for case report"
+* address 1..1
+* address.country 1..1
+* address.state 1..1
+
+Profile: WhoCrObservationAge
+Parent: Observation
+Id: who-cr-observation-age
+Title: "WHO CR Observation Age"
+Description: "WHO Observation of age for case report"
+* code = $LNC#30525-0
+* value[x] only Quantity
+* valueQuantity.value 1..1
+* valueQuantity from WhoCrValueSetAgeUnits
+
+Profile: WhoCrObservationBirthSex
+Parent: Observation
+Id: who-cr-observation-birth-sex
+Title: "WHO CR Observation Birth Sex"
+Description: "WHO Observation of birth sex for case report"
+* code = $LNC#76689-9
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetSexAtBirth
+
+Profile: WhoCrObservationLabTestDate
+Parent: Observation
+Id: who-cr-observation-lab-test-date
+Title: "WHO CR Observation Lab Test Date"
+Description: "WHO Observation for lab test date"
+* code = http://test.org/obsCode#lab-test-date // TODO: code not defined in questionnaire either
+* value[x] only dateTime
+* valueDateTime 1..1
+
+Profile: WhoCrObservationSymptoms
+Parent: Observation
+Id: who-cr-observation-symptoms
+Title: "WHO CR Observation Symptoms"
+Description: "WHO Observation if the patient has any symptoms for case report"
+* code = http://test.org/obsCode#symptoms // TODO: code not defined in questionnaire either
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNo
```

Comment:
The url associated with this value set code does not appear to resolve anywhere?
Is that okay? 
alias cs-v2-0136 -> http://terminology.hl7.org/CodeSystem/v2-0136

Reply:
I also noticed that. It might be just used as an identifier rather than pointing to somewhere real but don't really know.

Reply:
Just odd because the url alias below it does point to a real site :|
Therefore I thought it may be a typo. But I guess we don't have the spec for that